examples/angular/src/app/shared/models.ts

```diff
@@ -0,0 +1,78 @@
+import {
+    DotCMSBasicContentlet,
+    DotCMSComposedPageResponse,
+    DotcmsNavigationItem,
+    DotCMSPageAsset
+} from '@dotcms/types';
+
+export interface FileAsset {
+    fileAsset: {
+        versionPath: string;
+    };
+}
+
+export interface ContentletImage extends FileAsset {
+    identifier: string;
+    fileName: string;
```

Comment:
Can we separate the Contentlet Models in a `content.model.ts` file?

Reply:
Done!